app/src/main/java/com/example/stromtracker/ui/amortisationsrechnerPV/AmortisationsrechnerPVFragment.kt

```diff
@@ -42,8 +42,8 @@ class AmortisationsrechnerPVFragment : Fragment() {
         outJahresertragEuro= root.findViewById(R.id.amortPV_text_JahresertragEuro_zahl)
         outAmort = root.findViewById(R.id.amortPV_text_amortdauer_zahl)
```

Comment:
XML naming and ids should be in snake case only.

Reply:
This will be fixed in the Pull Request from AmortisationsrechnerPV.